marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployDatabaseFieldCommand.java

```diff
@@ -144,6 +144,9 @@ protected void addExistingRangePathIndexes(Fragment newProps, Fragment existingP
 
     protected void addExistingPathNamespaces(Fragment newProps, Fragment existingProps) {
         Element newNamespaces = newProps.getInternalDoc().getRootElement().getChild("path-namespaces", MANAGE_NS);
+        if(newNamespaces == null) {
```

Comment:
Can you add a comment here as to why newNamespaces may be null? I just reproduced this, am trying to find out why it happens.

Reply:
I'm trying this on dh-5-example project, and it appears that the new final-database.xml file is not being written to the project. Not sure what's up yet.

Reply:
Okay I see the problem now. We need a different fix, this PR can be closed.